artemis-server/src/main/java/org/apache/activemq/artemis/core/postoffice/impl/PostOfficeImpl.java

```diff
@@ -1744,6 +1746,15 @@ public void run() {
 
                   server.removeAddressInfo(address, null);
                }
+            } catch (ActiveMQAddressDoesNotExistException e) {
+               // the address and queue reaper is asynchronous so it may happen
+               // that the address is removed before the reaper removes it
+               ActiveMQServerLogger.LOGGER.debug(e.getMessage(), e);
+            } catch (ActiveMQShutdownException e) {
+               // the address and queue reaper is asynchronous so it may happen
+               // that the broker is shutting down while the reaper iterates
+               // through the addresses, next restart this operation will be retried
+               ActiveMQServerLogger.LOGGER.debug(e.getMessage(), e);
```

Comment:
log.debug please. same note as before

Reply:
I pushed a commit to replace the wrong logger.